src/main/resources/persistence.properties

```diff
@@ -4,7 +4,8 @@ hibernate.archive.autodetection = class
 
 javax.persistence.jdbc.url = jdbc:postgresql://188.166.92.29:5432/moodcat_db
 javax.persistence.jdbc.user = moodcat
-hibernate.hbm2ddl.auto = create-drop
+#hibernate.hbm2ddl.auto = create-drop
+hibernate.hbm2ddl.auto = update
```

Comment:
If there are no branches that go from this branch we could concider a rebase. And then squash the small commits and previous merges from master together. But it will mess up the reflog / history.